### ISSUE TITLE
SCRF Reference Template

### ISSUE THREAD

**Issue by isss111** [smartcontractresearchforum/discovery#317]
GDoc link: https://docs.google.com/document/d/1-Uerba9DFLGaqwWalMDY4smgoTA88lLSKBClDKG7chA/edit?usp=sharing

**Comment by isss111** [comment]
@marvino3 can you provide some clarity on the purpose of this template? Thanks.

**Comment by marvino3** [comment]
yes, we were looking to use this as a template for reference materials, such as the Web3 Glossary or Topic Taxonomy. I think this was our best guess for needing a template for reference materials but was never used or finalized. It would be good to check it against our references to see if it is relevant or if it can be retired or updated.

**Comment by isss111** [comment]
@ralphlombreglia this document is ready for review. Thank you.

**Comment by isss111** [comment]
@ralphlombreglia this document is ready for review. Thank you.
